src/C++/FieldConvertors.h

```diff
@@ -454,11 +444,7 @@ struct UtcTimeStampConvertor
     if( precision )
     {
       result[17] = '.';
-      if( integer_to_string_padded ( result + 18, precision, fraction )
```

Comment:
This does not need to be checked as the function integer_to_string_padded is now being tested by its own unit test in FieldConvertorsTestCase.cpp. This new unit test ensures the function does not produce the behavior being checked by this condition.